src/app/views/centres-and-facilities/centres-and-facilities.component.html

```diff
@@ -1,4 +1,4 @@
-<div style="height: 40vh; background-size: cover; background-image: url('https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTa3U1_-6QurYiV7Pd3a5F-DhYvfEM87eAtFKJHTGK4jh3ySppN2sTjb1qVWJNVKWlcVIw&usqp=CAU');">
+<div style="height: 20vh; background-size: cover;">
```

Comment:
Check the output difference due to this

Reply:
After removing the Background Image, it was looking too broad for just a line of text. So narrowed it down. Still, some problems need  to be fixed. Some, space on the right side is there.

![Screenshot 2024-03-26 171240](https://github.com/ICWaR/icwar.github.io/assets/138218874/acf71f29-d0a2-4aca-96a6-45a2676c26ac)

Reply:
So fix it first and then raise a pull request